packages/babel-parser/src/parser/lval.js

```diff
@@ -229,6 +229,49 @@ export default class LValParser extends NodeUtils {
     return exprList;
   }
 
+  isAssignable(node: N.Node, isBinding?: boolean): boolean {
+    switch (node.type) {
+      case "Identifier":
+      case "ObjectPattern":
+      case "ArrayPattern":
+      case "AssignmentPattern":
+        return true;
+
+      case "ObjectExpression": {
+        const last = node.properties.length - 1;
+        return node.properties.every((prop, i) => {
+          return (
+            prop.type !== "ObjectMethod" &&
+            (i === last || prop.type !== "SpreadElement") &&
```

Comment:
The bug that caused the regression was that this was using `=== "SpreadElement"` instead of `!==`.

Reply:
Should we also sync the changes to `flow` plugin?

Reply:
Why use 'every' ? It's slow vs a native for-loop